applications/frontend/src/store/reducers/workspaces.ts

```diff
@@ -0,0 +1,32 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+
+import { CallApiAction } from '../../middleware/backend'
+import { Workspace } from '../../types/workspace'
+
+const initialState: Workspace[] = [];
+
+const workspaceSlice = createSlice({
+    name: 'workspaces',
+    initialState,
+    reducers: {
+        loadWorkspaces(state, action: PayloadAction<Workspace[]>) {
```

Comment:
I see recurring this use of load* when only fetch* is defined. Am I missing something?

Reply:
explained